gensim/similarities/nmslib.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
```

Comment:
Why do you need this? I think this module does not need to be executable.

Reply:
Deleted the code. (I referred to index.py that implemented annoy indexer. index.py has this line.)